# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml
# Commit: 8c70bc92ad36
# Full Hash: 8c70bc92ad36beb61f2cd6031e80ace3793fe53b
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r c6f1243c3ba4 -r 8c70bc92ad36 gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml
--- a/gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml	Wed Feb 23 20:42:19 2022 +0000
+++ b/gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml	Wed Feb 23 20:49:26 2022 +0000
@@ -3,7 +3,7 @@
   items:
     - type: stacking-context
       transform-style: preserve-3d
-      transform: scale(-2, 44, 44727)
+      transform: scale(-2, 44, 1)
       items:
         - type: rect
           bounds: [ -100, -100, 200, 101 ]